api/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3-slim
 LABEL maintainer="Yash Ladha <yashladhapankajladha123@gmail.com>"
 
 
-ENV INSTALL_PATH /badgeyay
+ENV INSTALL_PATH /badgeyay/api
```

Comment:
This is tested on app engine? Have you tested the same in cloud as well?

Reply:
i've only tested it on the local docker container. it worked fine